### ISSUE TITLE
Support Slack webhook

### ISSUE THREAD

**Issue by tuha305** [jgillick/jira-changelog#24]
Hi there,
Thank for this amazing tool.
Slack's legacy tokens are deprecated so it would be great if you can add support for Slack's webhook.

Thanks!

**Comment by jgillick** [comment]
Good idea. This project isn't very high on my priority list these days, but I'd happily take a pull request with this update, if you wanted to try updating it.
